models/TransportationRequest.js

```diff
@@ -1,31 +1,6 @@
 const mongoose = require("mongoose");
 
 const TransportationRequestSchema = new mongoose.Schema({
-  dateTimeOFCall: {
-    type: Date,
-  },
-  callersName: {
-    type: String,
-  },
-  companyName: {
-    type: String,
-  },
-  dateOfAppointment: {
-    type: Date,
-  },
-  refferingProvider: {
-    type: String,
-  },
-  reasonForAppointment: {
-    type: String,
-  },
-  specialInsructions: {
-    type: String,
-  },
-  caseManagerName: {
-    type: Boolean,
-  },
-  // above ???????????
```

Comment:
Is it just me or does it look like we deleted a majority of the Transportation request schema? Don't we need this model and those data fields to accurately store instances of each ride request? Am I missing something?

Reply:
yes, I deleted unused data points. They were originally there based on an email you sent me a while ago, then when the frontend was completed I added in the data points based on what is sent, removing the this is just removing what was already unused

Reply:
Yeah, we still need those data points. 

All of the input that the end user completes to request a ride is needed for the model. Regardless of the shortened version that is sent via text message. Text message data is reduced for obvious reasons.

Let me know if the model instances being saved are still exactly the same as the ride request form data. If so, then we can merge this, otherwise, we need those data points back bruhhhh!